jmeter_run_all.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash +x
+
+# Scenario name is prefix added to the log files. So add something meaningful so that you can track the output easily. eg: saml_redirect_binding.
+scenario=$1
+
+# Location of the jmeter script.
+script_location=$2
+
+usage_msg="Usage: ./jmeter_run_all.sh <scenario_name> <jmeter_script_location>"
+usage_sample="Sample: ./jmeter_run_all.sh saml_redirect_binding /home/ubuntu/scripts/is_540_scripts/saml/SAML2-SSO-RedirectBinding.jmx"
+
+#### sanity check
+if [ -z "$1" ]
+  then
+  	echo "Please add a scenario name for the current test like, eg: saml_redirect_binding"
+	echo "$usage_msg"
+	echo "$usage_sample"
+	exit
+fi
+
+if [ -z "$2" ]
+  then
+        echo "Please provide a valid jmeter script location."
+	echo "$usage_msg"
+        echo "$usage_sample"
+	exit
+fi
+
+time_to_run=10800
+
+is_540_host="idp.wso2.com"
+is_540_host_username="ubuntu"
+
+carbon_home="/home/ubuntu/is_540/wso2is-5.4.0"
+java_home="/home/ubuntu/software/jdk1.8.0_144"
+jmeter_home="/home/ubuntu/software/apache-jmeter-3.3"
+output_directory="/home/ubuntu/output"
+private_key_path="/home/ubuntu/is540pf.pem"
```

Comment:
Looks like the system user is being picked above. In that case, when building the path shouldn't we incorporate the system user ?